src/pages/studyView/tabs/CNSegments.tsx

```diff
@@ -69,6 +69,12 @@ export default class CNSegments extends React.Component<{ store: StudyViewPageSt
         return this.activePromise.isComplete && (!this.activePromise.result || this.activePromise.result.length === 0);
     }
 
+    @computed get genome() {
+        const study = this.props.store.allPhysicalStudies.result?
+            this.props.store.allPhysicalStudies.result[0]: undefined;
+        return study? study.referenceGenome: DEFAULT_GENOME;
```

Comment:
i think there should be a space before ? and another before the colon in this ternary.

Reply:
if the reference genome is "global," meaning there is nothing specific to CNSegments, then it probably belongs in the store rather than here?

Reply:
agreed, `genome()` should be put in resultsviewpagestore